proxy/core/src/main/scala/io/cloudstate/proxy/AkkaSerializerRegisterFeature.scala

```diff
@@ -11,23 +11,56 @@ import java.util.concurrent.ConcurrentHashMap
 @AutomaticFeature
 final class AkkaSerializerRegisterFeature extends Feature {
   private[this] final val cache = ConcurrentHashMap.newKeySet[String]
+
+  // FIXME THIS IS CURRENTLY DISABLED SINCE IT WILL NOT MERGE ALL REFERENCE CONFS FOR SOME REASON
```

Comment:
@dwijnand @jroper See this—I never seem to be able to get all the Serializer config I need from the ClassLoader, so I think this AutomaticFeature is dead in the water. As such, in this PR, you see that I added the serializers to the native-image reflecting config files manually.